Makefile

```diff
@@ -0,0 +1,43 @@
+.PHONY: lint gather-submodules develop
+.DEFAULT_GOAL := help
+
+SHELL := /bin/bash
+export VIRTUALENVWRAPPER_PYTHON := /usr/bin/python
+
+define BROWSER_PYSCRIPT
+import os, webbrowser, sys
+
+try:
+	from urllib import pathname2url
+except:
+	from urllib.request import pathname2url
+
+webbrowser.open("file://" + pathname2url(os.path.abspath(sys.argv[1])))
+endef
+export BROWSER_PYSCRIPT
+
+define PRINT_HELP_PYSCRIPT
+import re, sys
+
+for line in sys.stdin:
+	match = re.match(r'^([a-zA-Z_-]+):.*?## (.*)$$', line)
+	if match:
+		target, help = match.groups()
+		print("%-20s %s" % (target, help))
+endef
+export PRINT_HELP_PYSCRIPT
+
+BROWSER := python -c "$$BROWSER_PYSCRIPT"
```

Comment:
All the code dealing with `BROWSER` looks unused. It should be removed if intentionally unused.